layouts/_default/baseof.html

```diff
@@ -89,6 +89,11 @@
         integrity="sha384-5f4X0lBluNY/Ib4VhGx0Pf6iDCF99VGXJIyYy7dDLY5QlEd7Ap0hICSSZA1XYbc4" crossorigin="anonymous"></script>
     {{ end }}
 
+    {{ if (.Params.mermaid) }}
+      <!-- MermaidJS support -->
+      <script src="https://cdn.jsdelivr.net/npm/mermaid/dist/mermaid.min.js"></script>
```

Comment:
It is a good practice to pin the version of our depedencies. So, maybe use `https://cdn.jsdelivr.net/npm/mermaid@9.1.3/dist/mermaid.core.min.js` here.

Reply:
ack